core/web/jobs_controller_test.go

```diff
@@ -477,7 +480,6 @@ targets:
 func TestJobsController_Create_WebhookSpec(t *testing.T) {
 	app := cltest.NewApplicationEVMDisabled(t)
 	require.NoError(t, app.Start(testutils.Context(t)))
-	t.Cleanup(func() { assert.NoError(t, app.Stop()) })
```

Comment:
nice - i've seen similar cleanup needed in helpers in mercury and llo. Maybe worth creating a ticket to audit (good for a new joiner?)